system-design/app-interoperability/observability.md

```diff
@@ -1,37 +1,40 @@
 # Application Observability
 
-Observability is about being able to use information outputs from a system to monitor how the system behaves internally.
+Observability involves the collection and analysis of information produced by a system to monitor its internal behavior.
 
 Observability data is captured using the following signals:
 
 - [Metrics](https://opentelemetry.io/docs/specs/otel/metrics/) - a numerical measurement in time used to observe change over a period of time or configured limits. For example, memory consumption, CPU Usage, available disk space.
-- [Logs](https://opentelemetry.io/docs/specs/otel/logs/) - text outputs produced by a running system/application to provide information about what is happing. For example, outputs to capture security events such as failed login attempts, or unexpected conditions such as errors.  
+- [Logs](https://opentelemetry.io/docs/specs/otel/logs/) - text outputs produced by a running system/application to provide information about what is happening. For example, outputs to capture security events such as failed login attempts, or unexpected conditions such as errors.  
 - [Traces](https://opentelemetry.io/docs/specs/otel/trace/) - contextual data used to follow a request's entire path through a distributed system. For example, trace data can be used to identify bottlenecks, or failure points, within a distributed system.
 
 Margo's application observability scope is limited to the following areas:
 
 - The device's container platform
 - The device's workload orchestration agent
-- The compliant containerized applications deployed to the device.
+- The compliant applications deployed to the device.
 
-The application observability data is intended to be used for the following purposes:
+The application observability data is intended to be used for purposes such as:
```

Comment:
Workload here as well.

Reply:
I'm unclear on the use of "application" vs "workload." Are you indicating we shouldn't be using "application" anywhere and instead should always use "workload"? If not, when should we use "workload" vs "application"?

Reply:
We agreed to defer this for now so there can be further discussion about using "workload" vs "application"